x/accounts/defaults/lockup/README.md

```diff
@@ -36,6 +53,133 @@ type BaseLockup struct {
 }
 ```
 
+
+### BaseClawback
+
+The base clawback account is used by all lockup accounts that are clawback enable. Like base lockup it contains the basic information for a lockup account. The Base clawback account keeps knowledge of the admin of the account, the admin is the wallet that will be able to trigger a clawback execution to claw the remain locked token in the lockup account. The BaseLBaseClawbackockup implemented BaseAccount interface. 
+
+```go
+type BaseClawback struct {
+	// Owner is the address of the account owner.
+	Owner collections.Item[[]byte]
+	// Admin is the address who have ability to request lockup account
+	// to return the funds
+	Admin           collections.Item[[]byte]
+	OriginalVesting collections.Map[string, math.Int]
+	WithdrawedCoins collections.Map[string, math.Int]
+	addressCodec    address.Codec
+	headerService   header.Service
+	// lockup end time.
+	EndTime collections.Item[time.Time]
+}
+```
+
+### Clawback enabled vs disabled
+
+Clawback are an extension to normal lockup account, the differances between a clawback enabled account and normal account is the BaseAccount interface implementer passed during account creation.
+
+Clawback enabled lockup account will be able to trigger a clawback execution by the admin to claw the remain locked token in the lockup account. Other execution still need to be triggered by the account owner. It will not be able to perform actions such as delegate, undelegate, or withdraw rewards like the default lockup account.
+
+```mermaid
+flowchart TD 
+    A[lockup account's BaseAccount] -->|type casting| B{Is BaseClawback?}
+
+    B -->|TRUE| C([Clawback enabled account])
+
+    B -->|FALSE| D([Clawback not enabled account])
+
+    C --> 1[Withdraw Unlocked Tokens]
+    C --> 2[Send Coins]
+    C --> 3[Clawback tokens]
+
+    D --> 1[Withdraw Unlocked Tokens]
+    D --> 2[Send Coins]
+    D --> 4[Delegate]
+    D --> 5[Undelegate]
+    D --> 6[Withdraw Reward]
```

Comment:
can a clawback account not delegate and withdraw rewards?

Reply:
it can not delegate and withdraw rewards

Reply:
why is that? then there are no rewards?